### PR TITLE
Use icons for source repos' users/stars

### PR DESCRIPTION
**Pull request by bl-ue**
See https://github.com/chubin/cheat.sh/pull/285#issuecomment-787416248

I also formatted the table so it is (it looks much better without word wrap)

### CODE REVIEW

**Comment by bl-ue** [comment]
Amusingly, the demo is available at the exact same URL :D

https://github.com/bl-ue/cheat.sh/tree/patch-1#cheat-sheets-sources

File: README.md [hunk 1]
```diff
@@ -820,15 +820,15 @@ and information sources, maintained by thousands of users, developers and author
...
+|-----------------------|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|-------------|
+|UNIX/Linux, programming|[cheat.sheets](https://github.com/chubin/cheat.sheets)                              |![](https://img.shields.io/github/contributors-anon/chubin/cheat.sheets?label=%F0%9F%91%A5&labelColor=white)           |![](https://img.shields.io/github/stars/chubin/cheat.sheets?label=%E2%AD%90&labelColor=white)           |May 1, 2017  |
+|UNIX/Linux commands    |[tldr-pages/tldr](https://github.com/tldr-pages/tldr)                               |![](https://img.shields.io/github/stars/tldr-pages/tldr?label=%F0%9F%91%A5&labelColor=white)                           |![](https://img.shields.io/github/stars/tldr-pages/tldr?label=%E2%AD%90&labelColor=white)               |Dec 8, 2013  |
```
Comment (bl-ue) [inline README.md:826]:
```suggestion
|UNIX/Linux commands    |[tldr-pages/tldr](https://github.com/tldr-pages/tldr)                               |![](https://img.shields.io/github/contributors-anon/tldr-pages/tldr?label=%F0%9F%91%A5&labelColor=white)                           |![](https://img.shields.io/github/stars/tldr-pages/tldr?label=%E2%AD%90&labelColor=white)               |Dec 8, 2013  |
```

**Comment by bl-ue** [comment]
I could also compress the table completely because that would probably look better wrapped, if you'd like.

**Comment by abitrolly** [comment]
Looks nice. Although Users title is a little misleading now, and I have no ideas. Maybe join the header cells - I don't know if markdown allows this, or it is HTML only option.

It will be even more useful if image links led to the contributors graph pages, like https://github.com/chubin/cheat.sh/graphs/contributors Right now links just open the image.

> I could also compress the table completely because that would probably look better when wrapped, if you'd like.

Not sure what the compressed table is.

**Comment by bl-ue** [comment]
>Not sure what the compressed table is.

I mean remove all of the extra whitespace that makes the source text for the table neater.  I added a bunch of whitespace to make the columns straight and aligned&mdash;but it looks really messy when it's wrapped because the resulting lines a really long...[see here](https://github.com/chubin/cheat.sh/pull/286/files).

>It will be even more useful if image links led to the contributors graph pages, like https://github.com/chubin/cheat.sh/graphs/contributors Right now links just open the image.

I thought of that, but the thing is, it would make the source text lines themselves even longer.  But if you don't mind, I'll gladly do it.

**Comment by chubin** [comment]
I agree that `Contributors` is better than `Users`; `Users` is actually completely wrong, and was added there just because of StackOverflow, because it is hard to say how many Contributors (i.e. users who answered at least one question) stackoverflow has. This information is definitely available somewhere, but I didn't find it on the spot.

I am goint to merge this. Thank you very much for your changes. Now it looks much better than before

**Pull request merged**